modules/virt-storage-pv-csi-overview.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * virt/storage/virt-storage-with-csi-paradigm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-storage-vp-csi-overview_{context}"]
+= Virtual machine CSI storage overview
+
+{VirtProductName} integrates with the Container Storage Interface (CSI) to manage VM storage. Storage classes define storage capabilities such as performance tiers and types. PersistentVolumeClaims (PVCs) request storage resources, which bind to PersistentVolumes (PVs). CSI drivers connect Kubernetes to vendor storage backends, including iSCSI, NFS, and Fibre Channel.
+
+image:virt-storage-csi-paradigm.png[title="Virtual machine disks and the CSI paradigm"]
```

Comment:
I would come back one day and add more descriptive alt text for users who can't see the image, but it's not a merge blocker.